.github/workflows/ci-build.yml

```diff
@@ -21,6 +21,26 @@ env:
   MEILISEARCH_PUBLIC_API_KEY: ${{ secrets.MEILISEARCH_PUBLIC_API_KEY }}
 
 jobs:
+  check-vulnerabilities:
+    name: Check third-party dependencies for vulnerabilities
+    runs-on: ubuntu-latest
+    steps:
+      - name: PyAnsys Vulnerability Check (on PRs)
+        if: github.event_name == 'pull_request'
+        uses: ansys/actions/check-vulnerabilities@v5
+        with:
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+          python-package-name: ${{ env.PACKAGE_NAME }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@RobPasMue We will have to figure out a work-around for the this token as it does not have permissions to create security advisories.

Reply:
Yep - I've got it. The PyAnsys CI bot token. But I didn't want to put it just yet. I just need to give admin privileges to the bot in this repo so that it can create advisories